backend/Api/Forecasts/ReadModelFactory.cs

```diff
@@ -0,0 +1,210 @@
+			.Select(projectGroup => new DetailedBookingForMonth(
+				BookingDetails.Staffing(projectGroup.Key, projectGroup.First(), bookingType),
+				Hours: months.Select(month => MonthlyHours.For(month, projectGroup.ToList(), consultant.Department.Organization)).ToList()));
+	}
+
+	private static IEnumerable<DetailedBookingForMonth> GetPlannedAbsences(Consultant consultant, List<DateOnly> months, List<Week> weeks)
+	{
+		return consultant.PlannedAbsences
+			.Where(absence => weeks.Contains(absence.Week))
+			.GroupBy(absence => absence.Absence.Name)
+			.Select(absenceGroup => new DetailedBookingForMonth(
+				BookingDetails.PlannedAbsence(absenceGroup.Key, absenceGroup.First()),
+				Hours: months.Select(month => MonthlyHours.For(month, absenceGroup.ToList(), consultant.Department.Organization)).ToList()));
+	}
+
+	// Using a similar pattern as in GetBookedHours() in StaffingController/ReadModelFactory
+	private static BookedHoursInMonth GetBookedHours(Consultant consultant, DateOnly month, IEnumerable<DetailedBookingForMonth> detailedBookings)
+	{
+		var organization = consultant.Department.Organization;
+
+		var totalHolidayHours = organization.GetTotalHolidayHoursInMonth(month);
+
+        var detailedBookingsArray = detailedBookings as DetailedBookingForMonth[] ?? detailedBookings.ToArray();
+
+        var totalBillable = DetailedBookingForMonth
+	        .GetTotalHoursForBookingTypeAndMonth(detailedBookingsArray, month, BookingType.Booking, evaluateBillable: true);
+
+        var totalNonBillable = DetailedBookingForMonth
+	        .GetTotalHoursForBookingTypeAndMonth(detailedBookingsArray, month, BookingType.Booking, evaluateBillable: true, isBillable: false);
+
+        var totalOffered = DetailedBookingForMonth
+	        .GetTotalHoursForBookingTypeAndMonth(detailedBookingsArray, month, BookingType.Offer);
+
+        var totalAbsence = DetailedBookingForMonth
+	        .GetTotalHoursForBookingTypeAndMonth(detailedBookingsArray, month, BookingType.PlannedAbsence);
+
+        var totalNotStartedOrQuit = DetailedBookingForMonth
+	        .GetTotalHoursForBookingTypeAndMonth(detailedBookingsArray, month, BookingType.NotStartedOrQuit);
+
+        var totalExcludableAbsence = detailedBookingsArray
+            .Where(s => s.BookingDetails is { Type: BookingType.PlannedAbsence, ExcludeFromBilling: true })
+            .Select(wh => wh.TotalHoursForMonth(month))
+            .Sum();
+
+        var totalVacations = DetailedBookingForMonth
+	        .GetTotalHoursForBookingTypeAndMonth(detailedBookingsArray, month, BookingType.Vacation);
+
+        var bookedTime = totalBillable + totalAbsence + totalVacations + totalHolidayHours + totalNonBillable + totalNotStartedOrQuit;
+        var forecastedBookedTime = bookedTime + totalOffered;
+
+        var workHoursInMonth = WorkloadHelper.CalculateWorkHoursInMonth(month, organization);
+
+        var totalSellableTime = Math.Max(workHoursInMonth - forecastedBookedTime, 0);
```

Comment:
```suggestion
        var totalSellableTime = Math.Max(workHoursInMonth - bookedTime, 0);
```

This was a misunderstanding. Scratch `forecastedBookedTime` and simply use `bookedTime` here

Reply:
Fixed in https://github.com/varianter/vibes/pull/605/commits/21bfa1f2f8241d97dbe8da5342da6023d2c0bbe4